README.md

```diff
@@ -17,11 +17,10 @@ Premonition will transform it into a markup block of your choice.
  * Non-intrusive - Content are presented as block-quotes by any other renderer.
  * Easy to install
  * Comes with a stylesheet (Sass/Css) and templates for rendering typical information boxes.
- * Support for both Kramdown and RedCarpet.
 
 ## Requirements
 
- * Jekyll 3.7.x or higher
+ * Jekyll 4.0.x or higher
```

Comment:
The `premonition.gemspec` only requires Jekyll >2.0. Should the requirement be made broader in the README, or narrower in the gemspec?

Reply:
Good question. I think narrower. Keeping backward compatibility is hard and not something I think we should spend time on.